packages/cli/src/databases/mysqldb/migrations/1648740597343-LowerCaseEmail.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import config = require('../../../../config');
+
+export class LowerCaseEmail1648740597343 implements MigrationInterface {
+	name = 'LowerCaseEmail1648740597343';
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		const tablePrefix = config.get('database.tablePrefix');
+
+		await queryRunner.query(`
+			UPDATE ${tablePrefix}user
+			SET email = LOWER(email);
+		`);
+	}
```

Comment:
Missing `logMigrationStart` and `logMigrationEnd`

Reply:
not needed for mysql since they run in a transaction and rollback if killed